src/style.css

```diff
@@ -1,3 +1,71 @@
+@import "https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap";
+
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --header-height: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  padding: 0;
+}
+
+.header__nav {
+  display: inline-block;
+}
+
+.header__list {
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.header__list-item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.header__list-item a {
+  position: relative;
+  display: inline-block;
+  line-height: var(--header-height);
+
+  color: #000;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  padding: 0;
+}
+
+a:hover, a.is-active{
```

Comment:
better to add the class to `a` tags and use class selectors

Reply:
Denys, I cant use classes, because, my extensions say`s that in tag <a> can be only 2 attributes, as you know tag <a> has href and some task say to add 'data-qa="hover"' attribute, i can`t add one more class then, so I decided to style tags, wihout class selectors.

Reply:
yet we need to use class for CSS styles) if you have any problems feel free to ask them in chat) I believe we will find the issue

Reply:
Okay, I will ask then)